### ISSUE TITLE
Enhancement: Served from Optimistic cache information.

### ISSUE THREAD

**Issue by cenk** [AdguardTeam/AdGuardHome#6742]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Is it possible to see "served from Optimistic Cache" information in the Response details page when the responses are served by the Optimistic Cache?

### Proposed solution

Is it possible to see "served from Optimistic Cache" information in the Response details page when the responses are served by the Optimistic Cache?

**Comment by cenk** [comment]
It would be very useful to be able to see this information without turning on verbose/debug mode. Just like "stale cache" information on the query page in "pi-hole".

**Comment by ainar-g** [comment]
So basically, the data you'd like to see is the number of cache misses due to TTL expiration?

**Comment by cenk** [comment]
No. Just "Served from Optimistic cache" information istead of "served from cache" for each response.

Correction: in pi-hole v6 (beta) version.

**Comment by ainar-g** [comment]
I'm sorry, but in that case what you're asking makes little sense.  There are no two separate common caches, there is only one that either has optimistic mode enabled or not.

**Comment by cenk** [comment]
Yes I know. There is only one cache. The name of the label may be different. 
May be "Served from cache (Optimistic response)"
May be "Served from cache (Expired)"

**Comment by yuxuan0107** [comment]
It would be better if you could provide a statistical display of the cache hit ratio

**Comment by yuxuan0107** [comment]
> It would be better if you could provide a statistical display of the cache hit ratio如果可以提供缓存命中率的统计显示，那就更好了

Of course, if optimistic caching is enabled, the statistics of the cache hit ratio may be inaccurate, so two statistics may be required, one is used to statistically display the hit ratio of the cache, and the other is used to statistically display the accuracy of the cached data in the optimistic cache

**Comment by ainar-g** [comment]
@yuxuan0107, cache statistics are a part of #1453.  Please keep discussions on-topic.

@cenk, I'm still not getting the feature request, sorry.  What is the problem you're trying to solve?  You've mentioned some kind of information that you want to track, but what information?  How would it be actionable?

**Comment by cenk** [comment]
@ainar-g, Can optimistic cache information be specified where I marked on this page? like "served from cache (optimistic response)"
![IMG_20240227_174621](https://github.com/AdguardTeam/AdGuardHome/assets/391865/87f40cc6-53af-499e-97ac-57d154d15fa6)
